<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GM_Util_ToolClass</class>
 <widget class="QMainWindow" name="GM_Util_ToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>586</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>GM_Util_Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="page">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageCharacterCreator">
       <widget class="QFrame" name="frameCharacterCreator">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>50</y>
          <width>521</width>
          <height>421</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <widget class="QPushButton" name="buttonSave">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>390</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Zapisz</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelCreatorName">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>10</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Kreator postaci&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QFrame" name="frameAvatar">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>40</y>
           <width>131</width>
           <height>131</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <widget class="QLabel" name="avatar">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>5</y>
            <width>121</width>
            <height>121</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="buttonAvatarChoice">
         <property name="geometry">
          <rect>
           <x>245</x>
           <y>72</y>
           <width>101</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>Wybierz awatar</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>180</y>
           <width>278</width>
           <height>201</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="layoutBackstory">
          <item row="0" column="0">
           <widget class="QLabel" name="labelBackstory">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Historia postaci&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextEdit" name="inputBackstory"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>36</y>
           <width>211</width>
           <height>354</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="layoutStats">
          <item row="5" column="2" colspan="3">
           <widget class="QLineEdit" name="inputArmor"/>
          </item>
          <item row="3" column="1" colspan="4">
           <widget class="QComboBox" name="inputProfession">
            <item>
             <property name="text">
              <string>Bard</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Czarodziej</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Druid</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Kapłan</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Kupiec</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Medyk</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Przestępca</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rzemieślnik</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Szlachcic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wiedźmin</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Zbrojny</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelProfession">
            <property name="text">
             <string>Klasa</string>
            </property>
           </widget>
          </item>
          <item row="10" column="2" colspan="3">
           <widget class="QLineEdit" name="inputWeaponName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelAge">
            <property name="text">
             <string>Wiek</string>
            </property>
           </widget>
          </item>
          <item row="11" column="2" colspan="2">
           <widget class="QLineEdit" name="inputWeaponDamage"/>
          </item>
          <item row="1" column="1" colspan="4">
           <widget class="QLineEdit" name="inputAge"/>
          </item>
          <item row="10" column="0" colspan="2">
           <widget class="QLabel" name="labelWeaponName">
            <property name="text">
             <string>Nazwa broni</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="5">
           <widget class="QCheckBox" name="checkBoxIsMagical">
            <property name="text">
             <string>Postać wrażliwa na magię</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelRace">
            <property name="text">
             <string>Rasa</string>
            </property>
           </widget>
          </item>
          <item row="11" column="4">
           <widget class="QLabel" name="labelK6">
            <property name="text">
             <string>k6</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2" colspan="3">
           <widget class="QLineEdit" name="inputAttack"/>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QLabel" name="labelEvasion">
            <property name="text">
             <string>Pot. uniku</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelHP">
            <property name="text">
             <string>HP</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2" colspan="3">
           <widget class="QLineEdit" name="inputSorcery">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="QLabel" name="labelSorcery">
            <property name="text">
             <string>Pot. Zaklęć</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelName">
            <property name="text">
             <string>Imię</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QComboBox" name="inputRace">
            <item>
             <property name="text">
              <string>Człowiek</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bobołak</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Elf</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gnom</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Krasnolud</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Niziołek</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Vran</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLabel" name="labelArmor">
            <property name="text">
             <string>Wyparowania</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="4">
           <widget class="QLineEdit" name="inputName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="2" colspan="3">
           <widget class="QLineEdit" name="inputEvasion"/>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QLabel" name="labelAttack">
            <property name="text">
             <string>Pot. ataku</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="4">
           <widget class="QLineEdit" name="inputHP"/>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="QLabel" name="labelWeaponDamage">
            <property name="text">
             <string>Obrażenia</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <zorder>labelCreatorName</zorder>
        <zorder>buttonSave</zorder>
        <zorder>layoutWidget</zorder>
        <zorder>frameAvatar</zorder>
        <zorder>buttonAvatarChoice</zorder>
        <zorder>layoutWidget_2</zorder>
       </widget>
       <widget class="QLabel" name="labelLoadTeam">
        <property name="geometry">
         <rect>
          <x>47</x>
          <y>490</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Wczytaj drużynę</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonLoadTeam">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>510</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Wczytaj</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="stackedWidgetPage2">
       <widget class="QFrame" name="frameCharacterCreator_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>70</y>
          <width>521</width>
          <height>421</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <widget class="QLabel" name="labelCreatorName_2">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>10</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QFrame" name="frameAvatar_2">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>40</y>
           <width>131</width>
           <height>131</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <widget class="QLabel" name="avatar_2">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>5</y>
            <width>121</width>
            <height>121</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget_3">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>180</y>
           <width>278</width>
           <height>201</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="layoutBackstory_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelBackstory_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Historia postaci&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextBrowser" name="displayBackstory"/>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>60</y>
           <width>21</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>181</width>
           <height>301</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="labelAge_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Wiek</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="labelRace_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Rasa</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="4">
           <widget class="QLabel" name="displayRace">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="labelProfession_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Klasa</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="4">
           <widget class="QLabel" name="displayClass">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="4">
           <widget class="QLabel" name="displayAge">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="labelHP_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>HP</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="4">
           <widget class="QLabel" name="displayHP">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <widget class="QLabel" name="labelArmor_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Wyparowania</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4" colspan="2">
           <widget class="QLabel" name="displayArmor">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="4">
           <widget class="QLabel" name="labelAttack_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Pot. ataku</string>
            </property>
           </widget>
          </item>
          <item row="5" column="4" colspan="2">
           <widget class="QLabel" name="displayAttack">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="4" colspan="2">
           <widget class="QLabel" name="displayEvasion">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="4">
           <widget class="QLabel" name="labelEvasion_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Pot. uniku</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="4">
           <widget class="QLabel" name="labelMagical">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Magiczny</string>
            </property>
           </widget>
          </item>
          <item row="7" column="4" colspan="2">
           <widget class="QLabel" name="displayMagical">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="4">
           <widget class="QLabel" name="labelSorcery_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Pot. Zaklęć</string>
            </property>
           </widget>
          </item>
          <item row="8" column="4" colspan="2">
           <widget class="QLabel" name="displaySorcery">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="4">
           <widget class="QLabel" name="labelWeaponName_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Nazwa broni</string>
            </property>
           </widget>
          </item>
          <item row="9" column="4" colspan="2">
           <widget class="QLabel" name="displayWeaponDamage">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="10" column="4" colspan="2">
           <widget class="QLabel" name="displayWeaponName">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="4">
           <widget class="QLabel" name="labelWeaponDamage_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Obrażenia</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>page</tabstop>
  <tabstop>buttonSave</tabstop>
  <tabstop>buttonAvatarChoice</tabstop>
  <tabstop>inputName</tabstop>
  <tabstop>inputAge</tabstop>
  <tabstop>inputRace</tabstop>
  <tabstop>inputProfession</tabstop>
  <tabstop>inputHP</tabstop>
  <tabstop>inputArmor</tabstop>
  <tabstop>inputAttack</tabstop>
  <tabstop>inputEvasion</tabstop>
  <tabstop>checkBoxIsMagical</tabstop>
  <tabstop>inputSorcery</tabstop>
  <tabstop>inputWeaponName</tabstop>
  <tabstop>inputWeaponDamage</tabstop>
  <tabstop>inputBackstory</tabstop>
 </tabstops>
 <resources>
  <include location="GM_Util_Tool.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBoxIsMagical</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelSorcery</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxIsMagical</sender>
   <signal>toggled(bool)</signal>
   <receiver>inputSorcery</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
